### PR TITLE
[PAL-512] Palmtree Scenario 2

### PR DESCRIPTION
**Pull request by Camji55**
https://tidepool.atlassian.net/browse/PAL-512

### CODE REVIEW

File: TidepoolOnboarding/Extensions/TPrescription.swift [hunk 1]
```diff
@@ -63,16 +60,18 @@ struct TDevices {
...
-        TDevices(pump: .coastal, cgm: .dexcomG6)
+        TDevices(pump: .palmtree, cgm: .dexcomG6)
         #endif
     }
     
     static func `for`(_ product: TidepoolSupport.Product) -> Self? {
         switch product {
         case .none: return nil
         case .palmtree: return TDevices(pump: .palmtree, cgm: .dexcomG6Demo)
+        case .palmtree1: return TDevices(pump: .palmtree, cgm: .dexcomG6Demo)
```
Comment (nhamming) [inline TidepoolOnboarding/Extensions/TPrescription.swift]: What is the difference between `palmtree` and `palmtree1`

**Review by nhamming — APPROVED** [review]

File: TidepoolOnboarding/Extensions/TPrescription.swift (hunk 1 above)
Comment (Camji55) [reply TidepoolOnboarding/Extensions/TPrescription.swift]: @nhamming the only difference is which scenarios are shown in the scenario selector. "palmtree" was configured to show everything while `palmtree1` should only show scenarios relating to that specific study product for the study. If all the scenarios are not needed in `palmtree` I can certainly remove the duplicate.

CleanShot 2024-04-15 at 10 21 10@2x

Comment (nhamming) [reply TidepoolOnboarding/Extensions/TPrescription.swift]: If you are up for it, maybe remove the duplicate.

Comment (Camji55) [reply TidepoolOnboarding/Extensions/TPrescription.swift]: Alright, duplicate removed!

**Pull request merged**